test/unit/video.test.js

```diff
@@ -603,3 +606,131 @@ QUnit.test('adds video-js class name with the video-js embed', function(assert)
   assert.ok(player.hasClass('video-js'), 'video-js class was added to the first embed');
   assert.ok(player2.hasClass('video-js'), 'video-js class was preserved to the second embed');
 });
+
+QUnit.module('videojs integration');
```

Comment:
can this be moved into a separate file, actually?

Reply:
sure